src/lib/icon/icon-module.ts

```diff
@@ -16,6 +16,5 @@ import {ICON_REGISTRY_PROVIDER} from './icon-registry';
   imports: [MatCommonModule],
   exports: [MatIcon, MatCommonModule],
   declarations: [MatIcon],
-  providers: [ICON_REGISTRY_PROVIDER],
```

Comment:
Where's the other provider?

Reply:
Yep
https://github.com/angular/material2/blob/master/src/lib/icon/icon-registry.ts#L95

Reply:
LGTM